### PR TITLE
updating the test cases

### PR DESCRIPTION
**Pull request by tierninho**
adds stashing, rebasing, next steps (wip), onboardin (wip), small edits

### CODE REVIEW

**Comment by tierninho** [comment]
@billygriffin Can I get an approval on this? Thanks.

File: docs/process/testing.md [hunk 1]
```diff
@@ -227,11 +232,22 @@
     - [ ] Merge hint shows status and branches to be merged
     - [ ] `Merge` button only activated if something to merge (includes awaiting conflicts)
       - [ ] If merged, success banner is shown temporarily; user can `X` to remove
-      - [ ] If conflicts, conflict modal shows quality of files, ability to open in [editor], open in command line, `Abort` button, while `Commit merge` button is deactivated until files are resolved; Changes tab shows all files 
+      - [ ] If conflicts, conflict modal shows quanity of files, ability to open in [editor], open in command line, `Abort` button, while `Commit merge` button is deactivated until files are resolved; Changes tab shows all files
```
Comment (say25) [inline docs/process/testing.md]:
```suggestion
      - [ ] If conflicts, conflict modal shows quantity of files, ability to open in [editor], open in command line, `Abort` button, while `Commit merge` button is deactivated until files are resolved; Changes tab shows all files 
```

File: docs/process/testing.md [hunk 2]
```diff
@@ -128,7 +130,8 @@
       - [ ] `Cancel` button does not save any changes made; modal closed
     - [ ] Appearance
       - [ ] Light theme is default
-      - [ ] Dark theme is optional      
+      - [ ] Dark theme is optional 
+      - [ ] For MAC OS, users can opt to match system prefence theme with checkbox
```
Comment (shiftkey) [inline docs/process/testing.md]:
```suggestion
      - [ ] For Mac, users can opt to match system preference theme with checkbox
```

File: docs/process/testing.md [hunk 3]
```diff
@@ -227,11 +232,22 @@
...
     - [ ] `Merge` button only activated if something to merge (includes awaiting conflicts)
       - [ ] If merged, success banner is shown temporarily; user can `X` to remove
-      - [ ] If conflicts, conflict modal shows quality of files, ability to open in [editor], open in command line, `Abort` button, while `Commit merge` button is deactivated until files are resolved; Changes tab shows all files 
+      - [ ] If conflicts, conflict modal shows quantity of files, ability to open in [editor], open in command line, `Abort` button, while `Commit merge` button is deactivated until files are resolved; Changes tab shows all files 
         - [ ] If conflict resolved, files marked green, and user can click `Commit merge` button
 	    - [ ] If merged, success banner is shown temporarily; user can `X` to remove
 	    - [ ] Binary files must be resolved in command line before committing merge
         - [ ] Aborting partially resolved commit surfaces "Are you sure?" dialogue; `Cancel` or `Abort merge` buttons
+  - [ ] Rebase Current Branch...
+    - [ ] Use can filter to find existing branches
```
Comment (billygriffin) [inline docs/process/testing.md]:
```suggestion
    - [ ] User can filter to find existing branches
```

File: docs/process/testing.md [hunk 4]
```diff
@@ -227,11 +232,22 @@
...
+    - [ ] User can select branch, other than current one
+    - [ ] Rebase hint shows status and branches to be merged
+    - [ ] `Start rebase` button only activated if something to rebase
+      - [ ] If rebase, success banner is shown temporarily; user can `X` to remove
+      - [ ] If conflicts, conflict modal shows quanity of files, ability to open in [editor], open in command line, `Abort` button, while `Continue rebase` button is deactivated until files are resolved
```
Comment (billygriffin) [inline docs/process/testing.md]:
```suggestion
      - [ ] If conflicts, conflict modal shows quantity of files, ability to open in [editor], open in command line, `Abort` button, while `Continue rebase` button is deactivated until files are resolved
```

File: docs/process/testing.md [hunk 5]
```diff
@@ -227,11 +232,22 @@
...
+        - [ ] If rebase resolved, files marked green, and user can click `Continue rebase` button
```
Comment (billygriffin) [inline docs/process/testing.md]:
```suggestion
        - [ ] If conflicts resolved, files marked green, and user can click `Continue rebase` button
```

File: docs/process/testing.md [hunk 6]
```diff
@@ -227,11 +232,22 @@
...
+	    - [ ] If rebased, success banner is shown temporarily; user can `X` to remove
+	    - [ ] Binary files must be resolved in command line before committing rebase
```
Comment (billygriffin) [inline docs/process/testing.md]: Is this accurate? Don't we have support for choosing binary files in desktop now?

File: docs/process/testing.md [hunk 7]
```diff
@@ -244,14 +260,18 @@
   - [ ] `Report Issue...` opens issue filing in Desktop repository on `github.com`
   - [ ] `Contact GitHub Support...` opens `https://github.com/contact` page with user and build prepopulated
   - [ ] `Show User Guides` opens Desktop help page on `github.com`
+  - [ ] `Show Keyboard Shurtcuts` opens `https://help.github.com/en/desktop/getting-started-with-github-desktop/keyboard-shortcuts-in-github-desktop`
```
Comment (billygriffin) [inline docs/process/testing.md]:
```suggestion
  - [ ] `Show Keyboard Shortcuts` opens `https://help.github.com/en/desktop/getting-started-with-github-desktop/keyboard-shortcuts-in-github-desktop`
```

File: docs/process/testing.md [hunk 8]
```diff
@@ -244,14 +260,18 @@
...
     - [ ] Mac: `ls ~/Library/Application\ Support/GitHub\ Desktop/Logs/*.log`
     - [ ] Windows: `%LOCALAPPDATA%\\Desktop\\*.desktop.production.log`
   - [ ] About GitHub Desktop (Windows only)
+  
+### Next Steps (wip)
 
 ### Repositories list
   - [ ] Current repository is always shown in top slot with respective icon; if repository exists
-  - [ ] Opening list shows all repositories in categorized alpha format with a working filter
+  - [ ] Opening list shows all repositories in categorized by owner in alpha format with a working filter
```
Comment (billygriffin) [inline docs/process/testing.md]:
```suggestion
  - [ ] Opening list shows all repositories, categorized by owner in alpha format with a working filter
```

File: docs/process/testing.md [hunk 9]
```diff
@@ -285,6 +305,11 @@
 	- [ ] User can open in finder, preferred editor, or OS default program  
   - [ ] Panes can be resized horizontally, and contents resize to take the full width
     - [ ] Quitting Desktop and relaunching remembers pane sizes
+  - [ ] Uncommited files are stashing if user attempts to switch branches
```
Comment (billygriffin) [inline docs/process/testing.md]:
```suggestion
  - [ ] Uncommitted files are optionally stashed if user attempts to switch branches
```

File: docs/process/testing.md [hunk 10]
```diff
@@ -285,6 +305,11 @@
...
+    - [ ] Modal asks user to stash on current branch or bring changes to new branch; `Cancel` or `Switch Branch` buttons
+      - [ ] If stashed then changes shown under Stashed Changes section of the under Changes tab
```
Comment (billygriffin) [inline docs/process/testing.md]:
```suggestion
      - [ ] If stashed then changes shown under Stashed Changes section below Changes tab when returning to the original branch
```

File: docs/process/testing.md [hunk 11]
```diff
@@ -285,6 +305,11 @@
...
+        - [ ] Stashed changes section show all stashed files; user can discard or restore to Changes
```
Comment (billygriffin) [inline docs/process/testing.md]:
```suggestion
        - [ ] Stashed changes section show all stashed files and diffs; user can discard or restore to Changes
```

File: docs/process/testing.md [hunk 12]
```diff
@@ -285,6 +305,11 @@
...
+      - [ ] If moved to new branch, files under Changes tab
```
Comment (billygriffin) [inline docs/process/testing.md]:
```suggestion
      - [ ] If moved to new branch, all previously changed files will exist under Changes tab on the new branch
```

File: docs/process/testing.md (hunk 6 above)
Comment (tierninho) [reply docs/process/testing.md]: You are correct. I had copy/pasted the merge conflict steps for rebasing. Fixing it.

**Comment by tierninho** [comment]
Updated with all the changes and added Onboarding and Next Steps

**Pull request merged**